scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -401,7 +403,7 @@ clrLine () {
 # Command    : clrScreen
 # Description: clear the screen, move to (0,0), and reset screen update counter
 clrScreen () {
-  clear
+  printf "\033[2J"
```

Comment:
@asnakep I have not run into instances where `clear` might not be available as both debian and enterprise linux based distros will have the libncurses or ncurses-libs packages installed from guild-deploy.sh which should ensure `clear` is always available.

That said I'm not against relying on bash `printf` for screen clears.  Although, it might be nice to add a note to the description mentioning the change and reasoning for others to consider.

Reply:
I didn't modify clrScreen function

Reply:
hmmm, strange. 
```suggestion
  clear
```